login-workflow/README.md

```diff
@@ -22,57 +32,42 @@ yarn add @brightlayer-ui/react-auth-workflow
-
-To use the example project as a starting point, read our [Sample Project Integration ](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/sample-project-integration.md) instructions.
-
-# Usage (Security State)
-
-After setup, you are now able to access various security actions and state from within your application. Importing `useSecurityActions` and `useSecurityState` allows you use these hooks as follows:
-
-```ts
-import { useSecurityActions, useSecurityState } from '@brightlayer-ui/react-auth-workflow';
-
-const securityActions = useSecurityActions();
-const securityState = useSecurityState();
-```
-
-The `securityActions` allows you to access actions related to user authentication and de-authentication. You can call `securityActions.onUserNotAuthenticated();` to un-authenticate (i.e., log user out) from the application.
+To integrate the package into an existing project, read our [Existing Project Integration](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/existing-project-integration.md) instructions. Even if you are starting from scratch, it may be useful for you to refer to the [Example](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/example) project while getting started.
 
-The `securityState` allows you to access state related to security, such as the currently authenticated user's email (`securityState.email`).
+In short, you will need to:
+-   Add routes for each of the Login Workflow screens you wish to include
+-   Define a mechanism for tracking the authenticated state of the current user
+-   Add a registration route for the registration workflow component
+-   Configure route guards / protected routes to ensure users only have access to specific routes when appropriate
+-   Define actions files to integrate the workflow screens with your back-end API
 
-More information about React Auth Workflow's exported objects can found in the [API](https://github.com/etn-ccis/blui-react-workflows/tree/master/login-workflow/docs/API.md) documentation.
+> If you are starting a brand new project, the easiest way to get started with the React Auth Workflow is to use the [BLUI CLI](https://www.npmjs.com/package/@brightlayer-ui/cli) and select the Authentication template as your starting point. This will create a brand new project that exactly mirrors the example project from this repository.
 
-# Routing
+## Learn More
 
-This library uses [React Router](https://reactrouter.com/) for routing. It includes a `<BrowserRouter>` wrapping the entire application, so in your application, you may skip providing a Router and just render `<Route>`s.
+-   [Setting Up Routing](/TODO)
+-   [Internationalization](/TODO)
+-   [Adding a Login Workflow](/TODO)
+-   [Adding a Registration Workflow](/TODO)
+-   [Defining Actions](/TODO)
```

Comment:
Suraj and I are covering the actions stuff within the context of the authentication and registration guides. Unless this is a link into the API and how each action will be used within the workflow?